src/main/archetype/ui.tests/README.md

```diff
@@ -0,0 +1,63 @@
+UI Test Project compatible with AEMaaCS
+=======================================
+
+[TODO]
+
+## Structure
```

Comment:
In the generated project, all the titles (##) are gone. Looks like the README file is being interpreted and the headings removed.